phpunit/class-wp-theme-json-test.php

```diff
@@ -811,16 +809,10 @@ function test_get_template_parts() {
 			)
 		);
 
-		$template_parts = $theme_json->get_template_parts();
+		$actual   = $theme_json->get_template_parts();
+		$expected = array( 'small-header' => array( 'area' => 'header' ) );
 
-		$this->assertEqualSetsWithIndex(
-			$template_parts,
-			array(
-				'small-header' => array(
-					'area' => 'header',
-				),
-			)
-		);
+		$this->assertEqualSetsWithIndex( $expected, $actual );
```

Comment:
Nit: I have an observation, it's hard to follow when `$expected` and `$actual` are both extracted to their own variables.

Reply:
Going to revert this change, it's not necessary for this to work.

What happened here was that the expected and actual values were inverted, hence the tests reported something I struggled to understand. Personally, I find it easier to have `expected`/`actual` variables for tests, as I can use them as anchors to navigate the rest of the code. I understand this is a style choice, though. :man_shrugging:

Reply:
I'm probably biased by the tests I saw so far in WordPress core, like:

https://github.com/WordPress/wordpress-develop/blob/master/tests/phpunit/tests/blocks/block.php